### ISSUE TITLE
Support Blob-only storage account for bindings

### ISSUE THREAD

**Issue by jeffhollan** [Azure/azure-webjobs-sdk-script#2371]
Expect this may be gated on v2 extensions where we can version storage? Either way today when trying to use a binding with Azure Blob Only accounts you get a runtime error on being unable to resolve the host name.  Hopefully we can support blob-only, at very least would be good to have a better error.  Realize this may in fact be two issues but wanted to see if already being tracked or solved and can create in appropriate repos from there.

**Comment by mpseidel** [comment]
+1 for a better error message. Luckily it took me only 15mins or so to find the note about the current limitation in the docs after running into the error :) Others might waste more time on debugging this.

**Comment by christopheranderson** [comment]
@jeffhollan - might want to split this into better error messages and the actual support. Actual support is probably not in scope for a bit, but we always make room for error messages/etc.

**Comment by paulbatum** [comment]
The error message issue is tracked here:
https://github.com/Azure/azure-webjobs-sdk/issues/1499)

PR to fix it about to be merged: 
https://github.com/Azure/azure-webjobs-sdk/pull/1510

As for supporting blob-only accounts, the issue is that the trigger relies on blob logs in the consumption plan and these accounts do not have this logging (or did not, last time we checked).

Workaround is to use event grid:
https://docs.microsoft.com/en-us/azure/event-grid/resize-images-on-storage-blob-upload-event
